core/src/main/java/com/novoda/merlin/Merlin.java

```diff
@@ -42,6 +46,16 @@ public void registerBindable(Bindable bindable) {
         registerer.registerBindable(bindable);
     }
 
+    public Observable<Boolean> getRxConnectionStatusObervable() {
```

Comment:
`getRxConnectionStatusObservable` (typo)

could just be `getConnectionStatus()` or `getConnectionStatusObservable`, dropping the rx.